src/index.html

```diff
@@ -7,8 +7,28 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a class="logo_container">
+        <img class="logo" src="images/logo.png"
+        alt="logo">
+      </a>
+      <nav class="nav_container">
+        <ul>
+          <li class="list_item-1"><a class="is-active" href="#">Apple</a></li>
+          <li class="list_item-2"><a href="#">Samsung</a></li>
+          <li class="list_item-3"><a href="#">Smartphones</a></li>
+          <li class="list_item-3" data-qa="hover"><a href="#">Laptops & Computers</a></li>
+          <li class="list_item-5"><a href="#">Gadgets</a></li>
+          <li class="list_item-6"><a href="#">Tablets</a></li>
+          <li class="list_item-7"><a href="#">Photo</a></li>
+          <li class="list_item-8"><a href="#">Video</a></li>
```

Comment:
Why you need all these classes(list_item-1...8)?
Better to use https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child